h2o-py/h2o/auth.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+"""
+Data class for SPNEGO authentication.
+
+:copyright: (c) 2016 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+try:
```

Comment:
kerberos should be optional, we need to make sure the code doesn't fail if kerberos is not installed

and produce an user-understandable error message

Reply:
added a nicer message, this fails only when user tries to explicitly import from h2.auth, does not influence any other imports